app/components/provider_interface/select_course_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: form_object, url: form_path, method: :post do |f| %>
```

Comment:
What is the plan for validating that at least one option is selected (and displaying an error summary)?

Reply:
We plan to test that on the form object, but not until we start working on the next card.